src-electron/db/query-impexp.js

```diff
@@ -674,17 +678,24 @@ WHERE
   } else if (reportingPolicy == dbEnums.reportingPolicy.prohibited) {
     attribute.reportable = false
   }
-
-  if (storagePolicy == dbEnums.storagePolicy.attributeAccessInterface) {
-    attribute.storageOption = dbEnums.storageOption.external
+  if (attributeId) {
+    forcedExternal = await queryUpgrade.checkGlobals(db, attributeId)
+    storageOption = await queryUpgrade.checkStorage(
+      db,
+      cluster.name,
+      null,
+      storagePolicy,
+      forcedExternal,
+      attributeId
```

Comment:
Since you just need the attribute name. Use the above query to retrieve the name as well.

LECT
  A.ATTRIBUTE_ID,
  A.REPORTING_POLICY,
  A.STORAGE_POLICY,
  A.NAME

Reply:
oh good idea, will do